crmd/join_client.c

```diff
@@ -226,7 +226,7 @@ do_cl_join_finalize_respond(long long action,
     }
 
     /* send our status section to the DC */
-    crm_debug("Confirming join join-%d: %s", join_id, crm_element_value(input->msg, F_CRM_TASK));
+    crm_debug("Confirming join join-%d", join_id);
```

Comment:
It's always CRM_OP_JOIN_ACKNAK here, and "Confirming join join-1: ack_nack" isn't particularly helpful :-) Looking at it again, there is a more useful message printed immediately after this, so I think I'll get rid of this and improve the wording on the others.